code/game/atoms.dm

```diff
@@ -915,9 +915,16 @@
  * Should be called through the [EX_ACT] wrapper macro.
  * The wrapper takes care of the [COMSIG_ATOM_EX_ACT] signal.
  * as well as calling [/atom/proc/contents_explosion].
+ *
+ * Returns TRUE by default, and behavior should be implemented on children procs on a per-atom basis. Should only return FALSE if we resist the explosion for any reason.
+ * We assume that the default is TRUE because all atoms should be considered destructible in some manner unless they explicitly opt out (in our current framework).
+ * However, the return value itself doesn't have any external consumers, it's only so children procs can listen to the value from their parent procs (due to the nature of the [EX_ACT] macro).
+ * Thus, the return value only matters on overrides of this proc, and the only thing that truly matters is the code that is executed (applying damage, calling damage procs, etc.)
```

Comment:
We should stack_trace if the return value isn't true/false from the caller, like we do for balloon alerts.

Reply:
i looked for a bit in balloon alert code and couldn't find what you were referring to, what line/file has the stack_trace?